dnf/yum/rpmtrans.py

```diff
@@ -28,6 +29,10 @@
 import sys
 import tempfile
 import traceback
+import warnings
+
+
+warnings.filterwarnings('once', category=DeprecationWarning)
```

Comment:
Please, can you remove this? Deprecation warnings are already filtered in dnf/__init__.py and this line basically means, that another one Deprecation warning will be displayed.